forge-gui/res/cardsfolder/upcoming/bite_down.txt

```diff
@@ -0,0 +1,7 @@
+Name:Bite Down
+ManaCost:1 G
+Types:Instant
+A:SP$ Pump | Cost$ 1 G | ValidTgts$ Creature.YouCtrl | AILogic$ PowerDmg | TgtPrompt$ Select target creature you control | SubAbility$ SoulsDamage | StackDescription$ None | SpellDescription$ Target creature you control deals damage equal to its power to target creature or planeswalker you don't control.
+SVar:SoulsDamage:DB$ DealDamage | AILogic$ PowerDmg | ValidTgts$ Creature.OppCtrl,Planeswalker.OppCtrl | TgtPrompt$ Select creature or planeswalker you don't control.  | NumDmg$ X | DamageSource$ ParentTarget | SpellDescription$ This creature deals damage equal to its power to any target.
```

Comment:
SpellDescription$ should match the oracle "to target creature or planeswalker you don't control."

Reply:
This doesn't look resolved, it shouldn't say any target?